dist/redhat/systemd.inc

```diff
@@ -0,0 +1,19 @@
+%define systemd_daemon_reload \
+if [ -x /usr/bin/systemctl ]; then \
+    /usr/bin/systemctl daemon-reload ||: \
+fi \
+%{nil}
+
+%undefine systemd_post
+%define systemd_post() %{expand:%systemd_daemon_reload}
+
+%undefine systemd_preun
+%define systemd_preun() \
+if [ $1 -eq 0 ] && [ -x /usr/bin/systemctl ]; then \
+    /usr/bin/systemctl disable %1 ||: \
+    /usr/bin/systemctl stop %1 ||: \
+fi \
+%{nil}
```

Comment:
Please avoid copied LGPL code, instead insert the commands directly in the %post, %postun, %preun, etc. sections.

Reply:
Well it's re-implemented from scratch after you mentioned LGPL issue: https://github.com/scylladb/scylla/pull/5605#pullrequestreview-345154690

Reply:
There's no need for a .inc file. You can just add the commands to the scriptlet.

Reply:
Dropped .inc file, replaced %systemd_* macros with systemctl commands.